### PR TITLE
Modified preload for PARAD_A driver updates

### PR DESCRIPTION
**Pull request by krosburg**
Added new fields to ParameterDefs for the new Seabird PAR sensor and added those fields to ParameterDictionary.

### CODE REVIEW

File: csv/ParameterDictionary.csv [hunk 1]
```diff
@@ -11,7 +11,7 @@ DOSTA,DICT22,,do_stable_sample,"PD7,PD10,PD11,PD12,PD16,PD863,PD197,PD964,PD835,
 "SP_TEST,CTD_CABLED",DICT25,,ctdbp_no_calibration_coefficients,"PD7,PD10,PD11,PD12,PD16,PD863,PD312,PD175,PD176,PD177,PD178,PD179,PD229,PD180,PD181,PD182,PD183,PD184,PD186,PD185,PD187,PD280,PD158,PD163,PD164,PD165,PD166,PD167,PD168,PD169,PD170,PD171,PD256,PD174,PD281,PD244,PD243,PD246,PD245,PD248,PD247,PD250,PD249,PD252,PD251,PD254,PD253,PD255,PD257,PD258",PD7,,Engineering,Calibration Data,
 "SP_TEST,CTD_CABLED",DICT26,,ctdbp_no_status,"PD7,PD10,PD11,PD12,PD16,PD863,PD312,PD93,PD202,PD263,PD121,PD122,PD120,PD199,PD200,PD201,PD204,PD203,PD264,PD265,PD2293",PD7,,Engineering,Status Data,
 "SP_TEST,CTD_CABLED",DICT27,,ctdbp_no_configuration,"PD7,PD10,PD11,PD12,PD16,PD863,PD312,PD211,PD218,PD219,PD220,PD221,PD222,PD223,PD213,PD215,PD216,PD217,PD224,PD267,PD225,PD2294,PD2295,PD2296,PD2297,PD2298,PD2299,PD2300",PD7,,Engineering,Configuration Data,
-PARAD_A,DICT28,,parad_sa_sample,"PD7,PD10,PD11,PD12,PD16,PD863,PD312,PD28,PD188,PD15,PD192",PD7,,Science,Data Products,
+PARAD_A,DICT28,,parad_sa_sample,"PD7,PD10,PD11,PD12,PD16,PD863,PD312,PD28,PD188,PD15,PD192,PD191,PD27,PD189,PD190,PD8087,PD8088,PD8089,PD320,PD1144,PD438,PD437",PD7,,Science,Data Products,
```
Comment (danmergens) [inline csv/ParameterDictionary.csv]: It's worth checking to see if the order of the dictionary is preserved in the Data Portal pull-down. If so, it's best to have the derived data products at the end to maintain consistency with the other streams.

**Comment by danmergens** [comment]
Please update `RELEASE_NOTES.md`

File: csv/ParameterDefs.csv [hunk 2]
```diff
@@ -131,6 +131,10 @@ BETA,PRESF,press_coeff_b,PD173,quantity,,float32,,1,-9999999,Pressure Coefficien
...
 PARAD_A,PARAD,par_counts_output,PD192,function,,float32,,µmol photons m-2 s-1,-99999999,Photosynthetically Active Radiation,4,,PFID83,"{""counts_output"":""PD188"",""a0"":""CC_a0"",""a1"":""CC_a1"",""Im"":""CC_Im""}",OPTPARW,downwelling_photosynthetic_photon_flux_in_sea_water,OPTPARW_L1,Photosynthetically Active Radiation (PAR) is the measure of the density of photons per unit area that are in the spectral range of light (400-700 nanometers) that primary producers use for photosynthesis.,Photosynthetically Active Radiation (400-700 nm),Science Data,L1
+PARAD_A,PARAD,par_measured,PD191,quantity,,float32,,µmol photons m-2 s-1,-99999999,Measured Photosynthetically Active Radiation,4,,,,OPTPARW,downwelling_photosynthetic_photon_flux_in_sea_water,OPTPARW_L1,Photosynthetically Active Radiation (PAR) is the measure of the density of photons per unit area that are in the spectral range of light (400-700 nanometers) that primary producers use for photosynthesis.,Photosynthetically Active Radiation (400-700 nm),Science Data,L1
```
[the remark this answers is not in the document]
Comment (krosburg) [reply csv/ParameterDefs.csv]: Updated to to be in order.

File: csv/ParameterDictionary.csv (hunk 1 above)
Comment (krosburg) [reply csv/ParameterDictionary.csv]: OOINet dropdown does not appear to reflect order of PD numbers in ParamDict. Time is first, remaining are alphabetical.

**Pull request merged**